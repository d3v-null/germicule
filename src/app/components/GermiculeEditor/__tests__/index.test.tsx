import React from 'react';
import { render } from '@testing-library/react';

import { GermiculeEditor } from '..';

describe('<GermiculeEditor  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<GermiculeEditor />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
