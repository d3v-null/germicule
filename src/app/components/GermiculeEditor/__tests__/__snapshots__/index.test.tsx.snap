// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GermiculeEditor  /> should match snapshot 1`] = `
<div
  class="react-json-view"
  style="font-family: monospace; cursor: default; background-color: rgba(0, 0, 0, 0); position: relative;"
>
  <div
    class="pretty-json-container object-container"
  >
    <div
      class="object-content"
    >
      <div
        class="object-key-val"
      >
        <span>
          <span
            style="display: inline-block; cursor: pointer;"
          >
            <div
              class="icon-container"
              style="display: inline-block; width: 17px;"
            >
              <span
                class="collapsed-icon"
              >
                <svg
                  fill="currentColor"
                  style="vertical-align: top; color: rgb(42, 161, 152); height: 1em; width: 1em; padding-left: 2px;"
                  viewBox="0 0 15 15"
                >
                  <path
                    d="M0 14l6-6-6-6z"
                  />
                </svg>
              </span>
            </div>
            <span
              style="display: inline-block; color: rgb(0, 43, 54); letter-spacing: 0.5px; font-style: none; vertical-align: top; opacity: 0.85;"
            >
              <span
                class="object-key"
              >
                <span
                  style="vertical-align: top;"
                >
                  "
                </span>
                <span>
                  root
                </span>
                <span
                  style="vertical-align: top;"
                >
                  "
                </span>
              </span>
              <span
                style="display: inline-block; margin: 0px 5px; color: rgb(0, 43, 54); vertical-align: top;"
              >
                :
              </span>
            </span>
            <span
              style="display: inline-block; cursor: pointer; font-weight: bold; color: rgb(0, 43, 54);"
            >
              {
            </span>
          </span>
        </span>
        <span
          class="brace-row"
        >
          <span
            style="display: inline-block; cursor: pointer; font-weight: bold; color: rgb(0, 43, 54); padding-left: 0px;"
          >
            }
          </span>
          <div
            class="object-meta-data"
            style="display: inline-block; padding: 0px 0px 0px 10px;"
          >
            <span
              class="copy-to-clipboard-container"
            >
              <span
                style="cursor: pointer; display: inline;"
              >
                <span
                  class="copy-icon"
                >
                  <svg
                    fill="currentColor"
                    preserveAspectRatio="xMidYMid meet"
                    style="vertical-align: top; color: rgb(38, 139, 210); font-size: 15px; margin-right: 3px; height: 1em; width: 1em;"
                    viewBox="0 0 40 40"
                  >
                    <g>
                      <path
                        d="m30 35h-25v-22.5h25v7.5h2.5v-12.5c0-1.4-1.1-2.5-2.5-2.5h-7.5c0-2.8-2.2-5-5-5s-5 2.2-5 5h-7.5c-1.4 0-2.5 1.1-2.5 2.5v27.5c0 1.4 1.1 2.5 2.5 2.5h25c1.4 0 2.5-1.1 2.5-2.5v-5h-2.5v5z m-20-27.5h2.5s2.5-1.1 2.5-2.5 1.1-2.5 2.5-2.5 2.5 1.1 2.5 2.5 1.3 2.5 2.5 2.5h2.5s2.5 1.1 2.5 2.5h-20c0-1.5 1.1-2.5 2.5-2.5z m-2.5 20h5v-2.5h-5v2.5z m17.5-5v-5l-10 7.5 10 7.5v-5h12.5v-5h-12.5z m-17.5 10h7.5v-2.5h-7.5v2.5z m12.5-17.5h-12.5v2.5h12.5v-2.5z m-7.5 5h-5v2.5h5v-2.5z"
                      />
                    </g>
                  </svg>
                </span>
              </span>
            </span>
            <span
              class="click-to-add"
              style="vertical-align: top;"
            >
              <span
                class="click-to-add-icon"
              >
                <svg
                  fill="currentColor"
                  preserveAspectRatio="xMidYMid meet"
                  style="vertical-align: top; display: inline-block; color: rgb(42, 161, 152); cursor: pointer; font-size: 15px; margin-right: 3px; height: 1em; width: 1em;"
                  viewBox="0 0 40 40"
                >
                  <g>
                    <path
                      d="m30.1 21.4v-2.8q0-0.6-0.4-1t-1-0.5h-5.7v-5.7q0-0.6-0.4-1t-1-0.4h-2.9q-0.6 0-1 0.4t-0.4 1v5.7h-5.7q-0.6 0-1 0.5t-0.5 1v2.8q0 0.6 0.5 1t1 0.5h5.7v5.7q0 0.5 0.4 1t1 0.4h2.9q0.6 0 1-0.4t0.4-1v-5.7h5.7q0.6 0 1-0.5t0.4-1z m7.2-1.4q0 4.7-2.3 8.6t-6.3 6.2-8.6 2.3-8.6-2.3-6.2-6.2-2.3-8.6 2.3-8.6 6.2-6.2 8.6-2.3 8.6 2.3 6.3 6.2 2.3 8.6z"
                    />
                  </g>
                </svg>
              </span>
            </span>
          </div>
        </span>
      </div>
    </div>
  </div>
</div>
`;
